packages/react-error-overlay/src/components/Collapsible.js

```diff
@@ -9,6 +9,7 @@
 
 /* @flow */
 import React, { Component } from 'react';
+import type { Element } from 'react';
```

Comment:
Can you please leave them typed as `React.Element` instead of `import type`? For explicitness (`Element` is a global in Flow that means DOM element).

Reply:
Without the import flow fails:
![image](https://user-images.githubusercontent.com/5604548/30118599-09481f80-9356-11e7-8dd8-d79adfa94c58.png)

Flow doc suggests to import React as a namespace with import * as React from 'react' instead of as a default with import React from 'react'.
https://flow.org/en/docs/react/components/
But I thought it was a big change and I decided not to do it.
Cheers

Reply:
I could use this syntax: React$Element.

Reply:
Oh, I guess you'd have to `import * as React from 'react'` at which point you'd bump into a false positive warning about accessing `PropTypes` in 15. Let's keep it this way then.

Reply:
Can you do `import type { Element as ReactElement } from 'react'`?